aodntools/timeseries_products/velocity_hourly_timeseries.py

```diff
@@ -122,14 +121,12 @@ def velocity_hourly_aggregated(files_to_agg, site_code, input_dir='', output_dir
         with xr.open_dataset(os.path.join(input_dir, file)) as nc:
             error_list = check_file(nc, site_code)
             if error_list:
-                bad_files.append([file, error_list])
-                rejected_files.append(file)
+                bad_files.update({file: error_list})
```

Comment:
I assumed that it's safe that file will neveer be overwritten by the update method correct?

Reply:
Yes. File names are unique.